web/src/components/organisms/Dashboard/index.tsx

```diff
@@ -68,7 +68,7 @@ const Dashboard: React.FC<Props> = ({ children, defaultSelectedKeys, menuType })
               />
             )}
           </DashboardSider>
-          <PaddedContent>{children}</PaddedContent>
+          <PaddedContent>{cloneElement(children, { handleWorkspaceModalOpen })}</PaddedContent>
```

Comment:
This Dashboard component is really starting to feel like a Frankenstein component. 

I already mentioned moving the menu to be a prop, but is there any other way you can rework this component to be a better common page layout wrapper? cloning the children and forcing handleWorkspaceModalOpen to ALL children whether they need it or not isn't really nice.